tensorflow/core/tpu/tpu_initializer_helper.cc

```diff
@@ -113,74 +113,69 @@ bool FindAndLogLibtpuProcess() {
-    // we're using different chips in different processes and thus multiple
-    // libtpu loads are ok.
-    // TODO(skyewm): we could make per-chip lock files and look at
-    // TPU_VISIBLE_DEVICES if we wanted to make this really precise.
-    std::string chips_per_process_bounds =
-        GetEnvVar("TPU_CHIPS_PER_PROCESS_BOUNDS");
-    bool allow_multiple_libtpu_load =
-        GetEnvBool("ALLOW_MULTIPLE_LIBTPU_LOAD", false);
-    // TODO(skyewm): remove this when TPU_CHIPS_PER_HOST_BOUNDS is fully
-    // deprecated
-    if (chips_per_process_bounds.empty()) {
-      chips_per_process_bounds = GetEnvVar("TPU_CHIPS_PER_HOST_BOUNDS");
-    }
-    if ((chips_per_process_bounds.empty() ||
-         chips_per_process_bounds == "2,2,1") &&
-        !allow_multiple_libtpu_load) {
-      int fd = open("/tmp/libtpu_lockfile", O_CREAT | O_RDWR, 0644);
-
-      // This lock is held until the process exits intentionally. The underlying
-      // TPU device will be held on until it quits.
-      if (lockf(fd, F_TLOCK, 0) != 0) {
-        if (!FindAndLogLibtpuProcess()) {
-          LOG(INFO) << "libtpu.so already in use by another process probably"
-                       " owned by another user. "
-                       "Run \"$ sudo lsof -w /dev/accel0\" to figure out "
-                       "which process is using the TPU. Not "
-                       "attempting to load libtpu.so in this process.";
-        }
-        should_load_library = false;
-      } else {
-        should_load_library = true;
+  // If TPU_CHIPS_PER_PROCESS_BOUNDS doesn't include all chips, we assume
+  // we're using different chips in different processes and thus multiple
+  // libtpu loads are ok.
+  // TODO(skyewm): we could make per-chip lock files and look at
+  // TPU_VISIBLE_DEVICES if we wanted to make this really precise.
+  std::string chips_per_process_bounds =
+      GetEnvVar("TPU_CHIPS_PER_PROCESS_BOUNDS");
+  bool allow_multiple_libtpu_load =
+      GetEnvBool("ALLOW_MULTIPLE_LIBTPU_LOAD", false);
+  // TODO(skyewm): remove this when TPU_CHIPS_PER_HOST_BOUNDS is fully
+  // deprecated
+  if (chips_per_process_bounds.empty()) {
+    chips_per_process_bounds = GetEnvVar("TPU_CHIPS_PER_HOST_BOUNDS");
+  }
+  if ((chips_per_process_bounds.empty() ||
+       chips_per_process_bounds == "2,2,1") &&
+      !allow_multiple_libtpu_load) {
+    int fd = open("/tmp/libtpu_lockfile", O_CREAT | O_RDWR, 0644);
+
+    // This lock is held until the process exits intentionally. The underlying
+    // TPU device will be held on until it quits.
+    if (lockf(fd, F_TLOCK, 0) != 0) {
+      auto pid = FindAndLogLibtpuProcess();
+      if (!pid.ok()) {
+        return errors::Aborted(
+            "libtpu.so already in use by another process probably  owned by "
+            "another user. Run \"$ sudo lsof -w /dev/accel0\" to figure out "
+            "which process is using the TPU. Not attempting to load "
+            "libtpu.so in this process.");
+      }
```

Comment:
You should run clang-format, I think there are some fixes (like `else` should be on this line)